packages/roosterjs-editor-core/lib/editor/Editor.ts

```diff
@@ -359,9 +383,33 @@ export default class Editor {
             content = extractContentEvent.content;
         }
 
+        if (this.core.inDarkMode && !getInDarkMode) {
+            content = this.getLightModeContent(content);
+        }
+
         return content;
     }
 
+    private getLightModeContent(content: string): string {

```

Comment:
Should this be in its own file? Likely, as it seems Editor.ts is designed for just the public stuff, and a private helper isn't in the right place.